common/components/internal-header/template.test.js

```diff
@@ -106,6 +111,17 @@ describe('Internal header component', function() {
         'my-attribute-2'
       )
     })
+
+    it('renders navigation item with text only', function() {
+      const $ = render('internal-header', examples['with navigation'])
```

Comment:
You could reuse the method used for the test above to pass in a custom set of items rather than using the ones from the example.

The idea behind the examples is to use those configurations to show examples of the component later on similar to how it's done on the [GOV.UK Design System](https://design-system.service.gov.uk/components/)